CHANGELOG.md

```
@@ -3,6 +3,7 @@
 #### 9.5.3 - SNAPSHOT
```

Markdownlint

Expected: h2; Actual: h4
Heading levels should only increment by one level at a time

(MD001, heading-increment)